.github/workflows/ci.yml

```diff
@@ -3,7 +3,9 @@ name: CI
 on:
   push:
     branches-ignore:
-      - "dependabot/**"
+      - gh-readonly-queue/**
+      - autodeps/**
+      - pre-commit-ci-update-config
```

Comment:
Any reason not to use string quoting? I've recently learned that yaml parsing is very, very strange when parsing things not marked as string literals

Reply:
I prefer no quotes where possible and use `>-` to force strings as it looks cleaner. But FWIW in this case, there's no chance YAML would parse cast these values as anything but string. We're safe here.